<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background: #1f2430;
	border: #c0c0c0;
	border-radius: 25px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txtIdUsuario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background: #f3f4f6;
	border: 1px solid #d0d0d0;
	border-radius: 20px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>  Buscar Empleado</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBuscar">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Medium</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid #fff;
	border-radius: 20px;
	color: #fff;;
}

QPushButton:hover{
	border-radius: 20px;
	background-color: rgb(77, 92, 107);
	color: #fff;
}</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tbwEmpleados">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
background: #83bac4;
border: 1px solid #D5D5D5;
border-radius: 15px;
}</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido P</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido M</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Correo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Telefono</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Puesto</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="txtRegistro">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>80</y>
      <width>390</width>
      <height>260</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit  {
	background: #83bac4;
	border: #c0c0c0;
	border-radius: 25px;
}</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Medium'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background: #1f2533;
	border-radius: 20px;
	color: #fff;
}

QPushButton:hover{
	border-radius: 20px;
	background-color: rgb(77, 92, 107);
	color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDatos">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>80</y>
      <width>400</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit  {
	background: #83bac4;
	border: #c0c0c0;
	border-radius: 25px;
}</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Medium'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>283</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background: #1f2533;
	border-radius: 20px;
	color: #fff;
}

QPushButton:hover{
	border-radius: 20px;
	background-color: rgb(77, 92, 107);
	color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>751</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame #frame_3{
	background: #83bac4;
	border: #c0c0c0;
	border-radius: 25px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>579</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Contrase√±a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>516</y>
        <width>100</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Nombre Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>299</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Correo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>390</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>460</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Puesto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Apellido Paterno:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>   Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPaterno">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Apellido Paterno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUsuario">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>505</y>
        <width>281</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nombre de Usuario</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtContrasenia">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>567</y>
        <width>290</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Contrase√±a</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEmail">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>319</y>
        <width>401</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Correo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTelefono">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>301</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Telefono</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbPuesto">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>446</y>
        <width>291</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SF UI Display SemBd</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>3</number>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	border: 1px solid #c0c0c0;
	border-radius: 5px;
	background: #fff;
	text-align: center;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Administrador</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Almacen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gerente general</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnRegistrar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>640</y>
        <width>161</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Medium</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #1f2533;
	border-radius: 20px;
	color: #fff;
}

QPushButton:hover{
	border-radius: 20px;
	background-color: rgb(77, 92, 107);
	color: #fff;
}</string>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Fecha Nacimiento:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateNacimiento">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>242</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
	border: 1px solid #c0c0c0;
	border-radius: 5px;
	background: #fff;
	text-align: center;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMaterno">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: 1px solid #c0c0c0;
	border-radius: 20px;
	background: #fff;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Apellido Materno</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
border: none;
}</string>
      </property>
      <property name="text">
       <string>Apellido Materno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>25</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>REGISTRO DE EMPLEADOS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>-30</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>recursos/Empleados.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVolver">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>640</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: #1f2430;
	border: 1px solid #e3c2a7;
	border-radius: 20px;
	color: #fff;
}

QPushButton:hover{	
	border-radius: 20px;
	background-color: rgb(77, 92, 107);
	color: #fff;
}
</string>
      </property>
      <property name="text">
       <string>VOLVER AL PANEL DE ACCESO</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>EMPLEADOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>txtRegistro</zorder>
   <zorder>frame_2</zorder>
   <zorder>tbwEmpleados</zorder>
   <zorder>btnLimpiar2</zorder>
   <zorder>txtDatos</zorder>
   <zorder>btnLimpiar</zorder>
   <zorder>frame</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
